app/models/ingredient.rb

```diff
@@ -0,0 +1,11 @@
+class Ingredient < ApplicationRecord
+  has_many :dish_ingredients
+  has_many :dishes, through: :dish_ingredients
+
+  def self.calorie_count
+    sum(:calories)
+  end
+
+  def self.chefs_ingredients
```

Comment:
Yeah, instead, we'd want to start this query from the Chef model, then it would be an easy query `ingredients.distinct`